MinecraftClient/Mapping/EntityTypeExtensions.cs

```diff
@@ -41,7 +46,9 @@ public static bool IsHostile(this EntityType e)
                 case EntityType.WitherSkeleton:
                 case EntityType.Zoglin:
                 case EntityType.Zombie:
+                case EntityType.ZombiePigman:
```

Comment:
Is there a difference between `ZombiePigman` and `ZombifiedPiglin` or did the name simply change in 1.6?
If so, it might be interesting to remove `ZombiePigman` from `EntityTypes` and use `ZombifiedPiglin` instead.
Does it make sense to you? Or should we keep both?

Reply:
They are the same mob but the name got changed. Let's just convert the old name to the new one.

Reply:
OK, doing this.